From e174cb4af7a47b891dcaa114f0a95fd4a64a9761 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 04 May 2020 11:09:20 +0000
Subject: [PATCH] GCC: remove noexcept on definition of move constructor

ScopedInterfaceEndpointHandle move constructor does not have noexcept specifier. However, noexcept can't be added there, because the move constructor uses new operator which can throw.

Bug: 819294
Change-Id: I61806b7aa6f1d77ed0668df7de5ef6cda5da97a5
---

diff --git a/third_party/blink/public/platform/cross_variant_mojo_util.h b/third_party/blink/public/platform/cross_variant_mojo_util.h
index a3a6d80..bb1095a 100644
--- a/third_party/blink/public/platform/cross_variant_mojo_util.h
+++ b/third_party/blink/public/platform/cross_variant_mojo_util.h
@@ -127,8 +127,8 @@
   CrossVariantMojoAssociatedReceiver() = default;
   ~CrossVariantMojoAssociatedReceiver() = default;
 
-  CrossVariantMojoAssociatedReceiver(
-      CrossVariantMojoAssociatedReceiver&&) noexcept = default;
+  CrossVariantMojoAssociatedReceiver(CrossVariantMojoAssociatedReceiver&&) =
+      default;
   CrossVariantMojoAssociatedReceiver& operator=(
       CrossVariantMojoAssociatedReceiver&&) noexcept = default;
 
@@ -160,8 +160,8 @@
   CrossVariantMojoAssociatedRemote() = default;
   ~CrossVariantMojoAssociatedRemote() = default;
 
-  CrossVariantMojoAssociatedRemote(
-      CrossVariantMojoAssociatedRemote&&) noexcept = default;
+  CrossVariantMojoAssociatedRemote(CrossVariantMojoAssociatedRemote&&) =
+      default;
   CrossVariantMojoAssociatedRemote& operator=(
       CrossVariantMojoAssociatedRemote&&) noexcept = default;
 
